README_VERSION.md

```diff
@@ -0,0 +1,19 @@
+# v1.1.3
+
+### What:
+
+> Fix
+
+resource "aws_security_group_rule" "ingress" replaced from count to for_each input.
+
+> Note
```

Comment:
Kindly add a short note that SG rules will be recreated for once as result of changing to for_each.